_topic_map.yml

```diff
@@ -2082,31 +2082,33 @@ Name: Service Mesh
 Dir: service_mesh
 Distros: openshift-enterprise,openshift-webscale
 Topics:
-#- Name: Service Mesh 2.x
-#  Dir: v2x
-#  Topics:
-#  - Name: Service Mesh 2.x release notes
-#    File: servicemesh-release-notes
-#  - Name: Service Mesh architecture
-#    File: ossm-architecture
-#  - Name: Service Mesh and Istio differences
-#    File: ossm-vs-community
-#  - Name: Preparing to install Service Mesh
-#    File: preparing-ossm-installation
-#  - Name: Installing Service Mesh
-#    File: installing-ossm
-#  - Name: Customizing the installation
-#    File: customizing-installation-ossm
-#  - Name: Deploying applications on Service Mesh
-#    File: prepare-to-deploy-applications-ossm
-#  - Name: Data visualization and observability
-#    File: ossm-observability
-#  - Name: Security
-#    File: ossm-security
-#  - Name: Traffic management
-#    File: ossm-traffic-manage
-#  - Name: Using the 3scale Istio adapter
-#    File: threescale-adapter
+- Name: Service Mesh 2.x
+  Dir: v2x
+  Topics:
+  - Name: Service Mesh 2.x release notes
+    File: servicemesh-release-notes
+  - Name: Service Mesh architecture
+    File: ossm-architecture
+  - Name: Service Mesh and Istio differences
+    File: ossm-vs-community
+  - Name: Preparing to install Service Mesh
+    File: preparing-ossm-installation
+  - Name: Installing Service Mesh
+    File: installing-ossm
+  - Name: Customizing the installation
+    File: customizing-installation-ossm
+  - Name: Deploying applications on Service Mesh
+    File: prepare-to-deploy-applications-ossm
+  - Name: Data visualization and observability
+    File: ossm-observability
+  - Name: Security
+    File: ossm-security
+  - Name: Traffic management
+    File: ossm-traffic-manage
+  - Name: Custom resources
```

Comment:
I think of the custom resources assembly as a reference topic, which is why I've pushed it down the toc. 

 @heathjoy, do you have an opinion?

Reply:
I'm thinking about how this scans if the customer downloads as PDF (I know, I'm a dinosaur thinking in a book model and not hypertext).  But since we know that the out-of-the-box configuration is just for getting started and isn't for production, I think we can assume that everyone is going to need to customize their deployment.  So I think it makes sense to keep the customizing the control plane and the CR reference close together in the docs.

Reply:
I see Julie's point but personally, I think we should stick with consistency for site navigation and search over PDF.

Reply:
@heathjoy, I think I need some context to understand your comment.  What are we being consistent with?  

I wasn't saying we should change the new reference topic that Neal has created in this PR.  I was suggesting that he move the reference topic to be immediately after the configuring topic in the table of contents.